libraries/cli/include/cli/default_config.hpp

```diff
@@ -16,6 +16,8 @@ const char *default_json = R"foo({
   "network_sync_level_size": 25,
   "network_packets_processing_threads": 14,
   "network_peer_blacklist_timeout" : 600,
+  "network_peer_light_node_timeout" : 600,
+  "is_light_node" : false,
```

Comment:
are we going to use also `light_node_history` config parameter together with `is_light_node` flag ?

- is_light_node is just easy way to say for most of the users if they wanna run light/full node through true/false flag
- light_node_history would be set to some default value that most of the users dont need to touch at all, but if someone builds an app on top of our blockchain and needs to run local node with lets say 1 week of history, he would have a possibility to set custom history for his light node